create_framework/introduction.rst

```diff
@@ -109,9 +109,8 @@ start with the simplest web application we can think of in PHP::
 
     printf('Hello %s', $input);
 
-If you have PHP 5.4, you can use the PHP built-in server to test this great
-application in a browser (``http://localhost:4321/index.php?name=Fabien``).
-Otherwise, use your own server (Apache, Nginx, etc.):
+You can use the PHP built-in server to test this great
+application in a browser (``http://localhost:4321/index.php?name=Fabien``):
```

Comment:
We should fix the position of the line break now (move a few words on the previous line).